From: Johannes Thumshirn <jthumshirn@suse.de>
Date: Thu, 3 Aug 2017 11:28:47 +0200
Subject: nvme-rdma: remove NVME_RDMA_MAX_SEGMENT_SIZE
Patch-mainline: v4.14-rc1
Git-commit: 4897ad4e0812c358c8bdd7aa25fdc2201ae2de0b
References: FATE#323952, FATE#322506

NVME_RDMA_MAX_SEGMENT_SIZE is not used anywhere, zap it.

Signed-off-by: Johannes Thumshirn <jthumshirn@suse.de>
Reviewed-by: Max Gurtovoy <maxg@mellanox.com>
Signed-off-by: Christoph Hellwig <hch@lst.de>
---
 drivers/nvme/host/rdma.c |    2 --
 1 file changed, 2 deletions(-)

--- a/drivers/nvme/host/rdma.c
+++ b/drivers/nvme/host/rdma.c
@@ -37,8 +37,6 @@
 
 #define NVME_RDMA_CONNECT_TIMEOUT_MS	3000		/* 3 second */
 
-#define NVME_RDMA_MAX_SEGMENT_SIZE	0xffffff	/* 24-bit SGL field */
-
 #define NVME_RDMA_MAX_SEGMENTS		256
 
 #define NVME_RDMA_MAX_INLINE_SEGMENTS	1
